Projects/Foundations/Schemas/Docs-Topics/S/Sparta/Sparta-Data-Mine/sparta-data-mine-004-relative-strength-index-rsi-sparta-indicator.json

```diff
@@ -0,0 +1,194 @@
+{
+    "topic": "Sparta Data Mine",
+    "pageNumber": 4,
+    "type": "Relative Strength Index (RSI) - Sparta Indicator",
+    "definition": {
+        "text": "The indicator produces the standard 14 setting of the Relative Strength Index.",
+        "updated": 1611662214310,
+        "translations": [
+            {
+                "language": "RU",
+                "text": "Индикатор выдает стандартную 14 настройку индекса относительной силы.",
+                "updated": 1643529696008
+            },
+            {
+                "language": "TR",
+                "text": "Özet: Gösterge, Göreceli Güç Endeksi'nin standart 14 ayarını üretir.",
```

Comment:
Didn`t this page exist before or did you just add it back after it got deleted? :+1:

Reply:
All files look great, when you answered and everything seems fine I will merge. :+1:

Reply:
Two of the same files appear in my repository. But I don't know why. Is this an issue that needs to be fixed?

Reply:
Are they in the same folder? If yes and they have the exact same filename and content you could delete one and push again, that could fix the problem. If the names or paths are somehow different please let me know as we had that problem before a couple times. :+1:

Reply:
Yes, they are in the same folder and with the same names. I deleted the untranslated file.

Reply:
The file is still showing up. Or was the other file not correct?

Reply:
The other file is not correct.

Reply:
But now everything is correct from your side? :+1:

Reply:
yes, you can combine